ko/admin/control.rst

```diff
@@ -201,11 +189,11 @@ CUBRID 서비스
     [service]
 
     # The list of processes to be started automatically by 'cubrid service start' command
-    # Any combinations are available with server, broker, manager, javasp and heartbeat.
+    # Any combinations are available with server, broker, manager and heartbeat.
```

Comment:
```suggestion
    # Any combination of  server, broker, manager and heartbeat is available.
```

Reply:
Thank you. but IMO, it is not the scope of this review.